setup/azurecmdfiles.wxi

```diff
@@ -1556,9 +1556,6 @@
                         <Component Id="cmpC67F14F377421F1BCD74EC5A579D43CB" Guid="*">
                             <File Id="filE9553CEC1D67AB3D4B77759239D60B18" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.IotHub\Microsoft.Azure.Commands.IotHub.dll-Help.xml" />
                         </Component>
-                        <Component Id="cmpD844DA64698210481D550B5CD8799D26" Guid="*">
-                            <File Id="filA23D9B350E9B87D17B90B2181AE71E4B" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.IotHub\Microsoft.Azure.Commands.IotHub.format.ps1xml" />
```

Comment:
Is this an intentional change?

Reply:
This is a change to fix broken build. This change - https://github.com/Azure/azure-powershell/pull/4503 removed a file, but this file was not updated.